castle/cms/browser/content/parallax.py

```diff
@@ -0,0 +1,74 @@
+from lxml import etree
+from plone.app.uuid.utils import uuidToObject
+from Products.Five import BrowserView
+from urlparse import urlparse, parse_qs
+
+
+class ParallaxView(BrowserView):
+
+    def get_tiles(self):
+        parallax_tiles = []
+        root = etree.fromstring(self.context.content)
+        tiles = root.findall('.//{http://www.w3.org/1999/xhtml}div[@data-tile]')
+        for tile in tiles:
+            tile_path = tile.get('data-tile')
+            if 'castle.cms.parallaxtile/' in tile_path:
+                parsed = urlparse(tile_path)
+                data = parse_qs(parsed.query)
+                parallax_tile = {}
+
+                # Parallax tile properties
+                try:
+                    image = uuidToObject(data['image:list'][0])
+                    parallax_tile['image'] = image.absolute_url()
+                except Exception:
+                    parallax_tile['image'] = None
+
+                parallax_tile['bg_color'] = data.get('bg_color', ['white'])[0]
```

Comment:
no action required here, but in the next iteration, we might allow users to type a hex code or css-recognized color (white, black, palegoldenrod, etc) in a textline with a css color validator constraint function to validate the field, and a default of 'black' or something, and similar with the text color. That should be a fairly straightforward mod if users seem interested in that.